SECURITY.md

```diff
@@ -0,0 +1,32 @@
+# Juno bug reporting and feature requests
+
+The Juno core development team uses GitHub to manage feature requests and bugs. This is done via GitHub Issues.
+
+## Triage and progress 🔜
+
+Issues added to GitHub will be triaged as they come in.
+
+Tracking of in-flight issues will be done through the Juno Core project board, but of course we reserve the right to not make a public issue if there is a security implication in doing so.
+
+## Feature request 🚀
+
+For a feature request, e.g. module inclusion, please make a GitHub issue. Clearly state your use case and what value it will bring to other users or developers on Juno.
```

Comment:
```suggestion
For a feature request, e.g. module inclusion, please make a GitHub issue. Clearly state your use-case and what value it will bring to other users or developers on Juno.
```